cli/src/commands/init/index.ts

```diff
@@ -365,6 +390,23 @@ export default class Init extends BaseCommand<typeof Init> {
     this.log();
   }
 
+  async promptForNextVersion(packageManager: PackageManager, pkg: string) {
+    const { command, args } = packageManager;
+    const { installNextVersion } = await this.prompt({
+      type: 'confirm',
+      name: 'installNextVersion',
+      message: `You are working with a Postgres enabled branch. We recommend using the 'next' version of the Xata SDK. Do you want to install the recommended version (${pkg}@next)?`
+    });
+    if (installNextVersion) {
+      await this.runCommand(command, [...args.split(' '), `${pkg}@next`]);
+      this.log();
+    } else {
+      await this.runCommand(command, [...args.split(' '), `${pkg}`]);
+      this.warn(`Please install ${pkg}@next later`);
+    }
+    return Boolean(installNextVersion);
```

Comment:
More a philisophical question:

Should we be asking? Or should we just say something like:

"We installed @next because you're on a Postgres enabled branch"

Reply:
I think it makes sense to download next if they're on a pgroll branch and there is no SDK version defined, but if they have a version defined we should always ask before overriding. I think it would be a relatively simple fix to this PR